bluebrain/deployment/gitlab-ci.yml

```
@@ -386,6 +386,20 @@ check_python_modules:
   script:
     - spack-check "$DEPLOYMENT_ROOT/config/modules.sh"
 
+# If the NEURON recipe has been modified then trigger the NEURON CI
+trigger_neuron_ci:
```

```suggestion
check_neuron_ci:
```
we have `check_modules` and `check_python_modules`, this would be more in                                             line with that.

---

OK, I'll push the update to the name at the same time as removing the neuron change (to test that we don't *always* trigger NEURON).